src/Maintenance/Maintenance/ChangeLog.md

```diff
@@ -20,6 +20,9 @@
 
 ## Upcoming Release
 
+## Version 1.2.0
```

Comment:
Remove this line. You just need to put notes under section upcoming release

Reply:
Fixed